README.md

```diff
@@ -0,0 +1,134 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [BookStore] <a name="about-project"></a>
+
+**BookStore** is an App that keeps track of activities. It allows users to create and manage a list of tasks or activities that they need to complete throughout the day or week.
+## 🛠 Built With<a name="built-with"></a>
+
+React
+
+### Tech Stack <a name="tech-stack"></a>
+
+Fullstack developer: includes both front end and Backend developer.
+
+### Key Features <a name="key-features"></a>
+
+- **Adding items.**
+- **Removing items.**
+- **Mark selected item as complete.**
+
+- **[Linters]**
+- **[Node]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo: (https://luchy-b.github.io/BookStore/dist/)
+
```

Comment:
- [ ] Please make sure that you structure this  `live demo` section well to look like the way it is in the [readme template](https://github.com/microverseinc/readme-template) :pray: